data/json/harvest.json

```diff
@@ -48,5 +48,33 @@
       { "drop": "sinew", "base_num": [ 50, 100 ], "scale_num": [ 0.6, 0.9 ], "max": 120 },
       { "drop": "raw_fur", "base_num": [ 2, 4 ], "scale_num": [ 0.6, 0.8 ], "max": 5 }
     ]
+  },
+  {
+    "id": "acidant_med",
+    "type": "harvest",
+    "message": "You carefully carve open the carapace, avoiding the acrid corrosion.",
+    "entries": [
+      { "drop": "meat_tainted", "base_num": [ 1, 5 ], "scale_num": [ 0.3, 0.7 ], "max": 8 },
```

Comment:
Ants are tainted but queen isn't?

Reply:
Yes, the queen's ovipositor for egg-laying supercedes acidopore production, thus all the soldiers and workers have acidopores, while the queen has reproduction capability.

http://www.antnest.co.uk/intanatomy.html
http://articles.extension.org/pages/11124/fire-ant-stings

Essentially, the acid sacs are volatile and unstable, so upon death, it implodes and contaminates all the meat.